clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -345,6 +345,144 @@ check_sane_control_flow()
         if (!run_checker(memrefs, false))
             return false;
     }
+#ifdef UNIX
+    // Negative test (PC discontinuity): Transition from instr to kernel_xfer event
```

Comment:
Let's just use the new notation "correct"/"incorrect" which is clearer than "positive"/"negative".